k8s/install/services/newrelic-logs/Makefile

```
@@ -0,0 +1,27 @@
+export MANIFEST_FILE ?= "newrelic-logs.yaml.j2"
+export FLUENTD_CPU_LIMIT ?= 100m
+export FLUENTD_MEMORY_LIMIT ?= 256Mi
+export NR_LICENSE_B64 = $(shell printf "${NR_LICENSE_KEY}" | base64)
+
+install-fluentd: check-env
+	j2 ${MANIFEST_FILE} | kubectl apply -f -
+
+test: check-env
+	j2 ${MANIFEST_FILE}
+
+delete: check-env
+	j2 ${MANIFEST_FILE} | kubectl delete -f -
+
```

Might be helpful! Not required if you don't like it

```suggestion

diff: check-env
       j2 ${MANIFEST_FILE} | kubectl diff -f -
```